server/services/store/storetests/board_insights.go

```diff
@@ -0,0 +1,99 @@
+package storetests
+
+import (
+	"testing"
+
+	"github.com/mattermost/focalboard/server/model"
+	"github.com/mattermost/focalboard/server/services/store"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	testInsightsUserID1 = "user-id-1"
+)
+
+func StoreTestBoardsInsightsStore(t *testing.T, setup func(t *testing.T) (store.Store, func())) {
+	t.Run("GetBoardsInsights", func(t *testing.T) {
+		store, tearDown := setup(t)
+		defer tearDown()
+		getBoardsInsightsTest(t, store)
+	})
+}
+
+func getBoardsInsightsTest(t *testing.T, store store.Store) {
+	// creating sample data
+	teamID := testTeamID
+	userID := testUserID
+	newBab := &model.BoardsAndBlocks{
+		Boards: []*model.Board{
+			{ID: "board-id-1", TeamID: teamID, Type: model.BoardTypeOpen, Icon: "💬"},
+			{ID: "board-id-2", TeamID: teamID, Type: model.BoardTypePrivate},
+			{ID: "board-id-3", TeamID: teamID, Type: model.BoardTypeOpen},
+		},
+		Blocks: []model.Block{
+			{ID: "block-id-1", BoardID: "board-id-1", Type: model.TypeCard},
+			{ID: "block-id-2", BoardID: "board-id-2", Type: model.TypeCard},
+			{ID: "block-id-3", BoardID: "board-id-1", Type: model.TypeCard},
+			{ID: "block-id-4", BoardID: "board-id-2", Type: model.TypeCard},
+			{ID: "block-id-5", BoardID: "board-id-1", Type: model.TypeCard},
+			{ID: "block-id-6", BoardID: "board-id-2", Type: model.TypeCard},
+			{ID: "block-id-7", BoardID: "board-id-1", Type: model.TypeCard},
+			{ID: "block-id-8", BoardID: "board-id-2", Type: model.TypeCard},
+			{ID: "block-id-9", BoardID: "board-id-1", Type: model.TypeCard},
+			{ID: "block-id-10", BoardID: "board-id-3", Type: model.TypeCard},
+			{ID: "block-id-11", BoardID: "board-id-3", Type: model.TypeCard},
+			{ID: "block-id-12", BoardID: "board-id-3", Type: model.TypeCard},
+		},
+	}
+
+	bab, err := store.CreateBoardsAndBlocks(newBab, userID)
+	require.Nil(t, err)
+	require.NotNil(t, bab)
+
+	newBab = &model.BoardsAndBlocks{
+		Blocks: []model.Block{
+			{ID: "block-id-13", BoardID: "board-id-1", Type: model.TypeCard},
+			{ID: "block-id-14", BoardID: "board-id-1", Type: model.TypeCard},
+		},
+	}
+	bab, err = store.CreateBoardsAndBlocks(newBab, testInsightsUserID1)
+	require.Nil(t, err)
+	require.NotNil(t, bab)
+	bm := &model.BoardMember{
+		UserID:      userID,
+		BoardID:     "board-id-2",
+		SchemeAdmin: true,
+	}
+
+	_, _ = store.SaveMember(bm)
+
+	boardsUser1, _ := store.GetBoardsForUserAndTeam(testUserID, testTeamID)
+	boardsUser2, _ := store.GetBoardsForUserAndTeam(testInsightsUserID1, testTeamID)
+	// t.Run("team insights", func(t *testing.T) {
```

Comment:
Why is this commented out?

Reply:
https://github.com/mattermost/focalboard/pull/3596/commits/64cb8fba794845a115dbcdd61aa2a42d2c4b7196